backend/medtagger/api/tasks/serializers.py

```diff
@@ -0,0 +1,30 @@
+"""Module responsible for storage of serializers used in Tasks endpoints."""
+from flask_restplus import fields
+
+from medtagger.api import api
+from medtagger.definitions import LabelTool
+
+out__label_tag = api.model('Label Tag model', {
+    'key': fields.String(),
+    'name': fields.String(),
+    'actions_ids': fields.List(fields.Integer(),
+                               attribute=lambda label_tag: [action.id for action in label_tag.actions]),
+    'tools': fields.List(fields.String(), description='Available tools for Label Tag',
+                         enum=[tool.name for tool in LabelTool],
+                         attribute=lambda label_tag: [tool.name for tool in label_tag.tools]),
+})
+
+out__task = api.model('Task model', {
+    'task_id': fields.Integer(attribute='id'),
```

Comment:
Please rename it to the `id` (instead of `task_id`). This model returns Tasks, so `task_` prefix is not needed :)

Reply:
If we will use the Key as a reference, I think that frontend doesn't have to know id